modules/n1ql/pages/n1ql-language-reference/createscope.adoc

```diff
@@ -37,7 +37,7 @@ scope::
 Refer to {naming-for-scopes-and-collections}[Naming for Scopes and Collections] for restrictions on scope names.
 
 NOTE: If there is a hyphen (-) inside the bucket name or the scope name, you must wrap that part of the path in backticks ({backtick} {backtick}).
-For example, `default:{backtick}travel-sample{backtick}` indicates the `travel-sample` keyspace in the `default` namespace.
+For example, `default:{backtick}my-db{backtick}` indicates the `my-db` keyspace in the `default` namespace.
```

Comment:
Now it's a `my-db` keyspace? I'm a little confused

Reply:
What exactly is confusing here? As far as I can tell this note is just telling the user to wrap any keyspace names in backticks if there's a hyphen in them. I'll redo the example and wording as we can't use $DATABASE_NAME. I'll probably put these in a partial too as they appear a lot.

Reply:
These sections are being removed.